pvsite_datamodel/sqlmodels.py

```diff
@@ -264,3 +268,53 @@ class StatusSQL(Base, CreatedMixin):
     status_uuid = sa.Column(UUID(as_uuid=True), default=uuid.uuid4, primary_key=True)
     status = sa.Column(sa.String(255))
     message = sa.Column(sa.String(255))
+
+
+class InverterSQL(Base, CreatedMixin):
+    """Class representing the inverters table.
+
+    Each InverterSQL row represents an inverter identified in
+    a successful Enode link.
+
+    *Approximate size: *
+    4 clients * ~1000 sites each * ~1 inverter each = ~4000 rows
+    """
+
+    __tablename__ = "inverters"
+
+    inverter_uuid = sa.Column(UUID(as_uuid=True), default=uuid.uuid4, primary_key=True)
+    client_uuid = sa.Column(
+        UUID(as_uuid=True),
+        sa.ForeignKey("clients.client_uuid"),
+        nullable=False,
+        comment="The UUID for the client that controls this inverter",
+    )
+    site_uuid = sa.Column(
+        UUID(as_uuid=True),
+        sa.ForeignKey("sites.site_uuid"),
+        nullable=True,
+        comment="The UUID for the site that has this inverter",
+    )
+
+    client: ClientSQL = relationship("ClientSQL", back_populates="inverters")
+    site: SiteSQL = relationship("SiteSQL", back_populates="inverters")
+
+
+class EnodeTokenSQL(Base, CreatedMixin):
```

Comment:
Considering the API's deployment system, this makes sense.

Our consideration was just to make this deployment-agnostic, so that it would maintain the token in a serverless environment as well.

We'll remove this table.